### PR TITLE
Integrate current activities with eachother

### PR DESCRIPTION
**Pull request by lizard-52**
I have tied the login, list, and add/view/edit activities together. You can now login (account names are hardcoded for now) as well as view/add/edit items (synced with DB). Also included is the new version of the add/edit/view activity. That probably should have been it's own PR.

### CODE REVIEW

**Comment by lizard-52** [comment]
@bpaul2 your UI changes are here too.

**Review by waridh — APPROVED** [review]
This works!

File: code/app/src/main/java/com/example/sigma_blue/ItemList.java [hunk 1]
```diff
@@ -169,4 +170,16 @@ public List<Item> getList() {
     public void setSummaryView(TextView summaryView) {
         this.adapter.setSummaryView(summaryView);
     }
+
```
[the remark this answers is not in the document]
Comment (lizard-52) [reply code/app/src/main/java/com/example/sigma_blue/ItemList.java:173]: I think you wrote this bit

**Review by echeng-git — APPROVED** [review]
LGTM.

**Pull request merged**